recipe/meta.yaml

```diff
@@ -361,8 +364,8 @@ outputs:
       run:
         - python
         - {{ pin_subpackage(name, max_pin='x.x.x.x.x.x') }}
-        - flask-appbuilder 4.3.1
-        # flask-appbuilder[oauth] for 4.3.1
```

Comment:
The dependencies for `flask-appbuilder[oauth]` have not changed in 4.3.3:
https://github.com/dpgaspar/Flask-AppBuilder/blob/v4.3.3/setup.py#L72